### DISCUSSION TITLE
useAppConfig in server routes

### DISCUSSION THREAD

**Discussion by webintmercatus** [nuxt/nuxt#16621] [Ideas]
I have defined an endpoint in `/server/api`

It looks like this:
```
export default async (res) => {

  const config = useAppConfig()

  return config
}
```

Would be nice if I can use my app config within my server files :)

**Comment by justindasilva** [comment]
Can a mod delete this issue please?
I posted this mistakenly with a test Github account.

I've added another topic here: https://github.com/nuxt/framework/discussions/8204
